torch/csrc/cuda/Module.cpp

```diff
@@ -422,15 +422,6 @@ PyObject* THCPModule_cudaCachingAllocator_enable(
   END_HANDLE_TH_ERRORS
 }
 
-PyObject* THCPModule_cudaCachingAllocator_set_allocator_settings(
-    PyObject* _unused,
-    PyObject* env) {
-  HANDLE_TH_ERRORS
-  c10::CachingAllocator::setAllocatorSettings(THPUtils_unpackString(env));
-  Py_RETURN_NONE;
-  END_HANDLE_TH_ERRORS
```

Comment:
cc @albanD This binding style will return `RuntimeError` if error value is provided. But pybinding style will return `ValueError`. So I change some `TORCH_CHECK` into `TORCH_CHECK_VALUE` to unify invalid value error into `ValueError`.